### PR TITLE
add some packages from LXQt

### PR DESCRIPTION
**Pull request by librehat**

### CODE REVIEW

File: packages/libqtxdg/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://lxqt.github.io
+TERMUX_PKG_DESCRIPTION="Qt 5 implementation of freedesktop.org XDG specifications"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="Simeon Huang <symeon@librehat.com>"
+TERMUX_PKG_VERSION=3.7.1
+TERMUX_PKG_SRCURL="https://github.com/lxqt/libqtxdg/releases/download/${TERMUX_PKG_VERSION}/libqtxdg-${TERMUX_PKG_VERSION}.tar.xz"
+TERMUX_PKG_SHA256=477cbe76b3305071ff5f5bfa31dbcddcc51f3434b9ed75d91988219296d88a9b
+TERMUX_PKG_DEPENDS="qt5-qtbase"
+TERMUX_PKG_BUILD_DEPENDS="cmake, lxqt-build-tools, glib, qt5-qtsvg, qt5-qttools-cross-tools"
```
Comment (xeffyr) [inline packages/libqtxdg/build.sh:9]: `qt5-qtsvg` is a build-time dependency?

File: packages/libqtxdg/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}"
```
Comment (xeffyr) [inline packages/libqtxdg/build.sh:10]: Is there a reason for specifying CMAKE_INSTALL_PREFIX manually?

This is already specified in https://github.com/termux/termux-packages/blob/master/scripts/build/configure/termux_step_configure_cmake.sh and shouldn't be specified manually unless you are overriding the whole CMake configuration step.

File: packages/lxqt-build-tools/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://lxqt.github.io
+TERMUX_PKG_DESCRIPTION="Building tools required by LXQt project"
+TERMUX_PKG_LICENSE="BSD-3-Clause"
+TERMUX_PKG_MAINTAINER="Simeon Huang <symeon@librehat.com>"
+TERMUX_PKG_VERSION=0.9.0
+TERMUX_PKG_SRCURL="https://github.com/lxqt/${TERMUX_PKG_NAME}/releases/download/${TERMUX_PKG_VERSION}/${TERMUX_PKG_NAME}-${TERMUX_PKG_VERSION}.tar.xz"
+TERMUX_PKG_SHA256=9782878d6c0cc896d4a54fce5cc322561cdf1df270b8f941b2088a434b535e4d
+TERMUX_PKG_BUILD_DEPENDS="cmake, qt5-qtbase"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}"
```
Comment (xeffyr) [inline packages/lxqt-build-tools/build.sh:9]: Same as libqtxdg.

File: packages/libqtxdg/build.sh (hunk 1 above)
Comment (librehat) [reply packages/libqtxdg/build.sh]: Moved to runtime. Their own read me said it is build time dependency...

**Comment by librehat** [comment]
Actually the lxqt-build-tools should be platform independent (no binaries)

**Review by xeffyr — APPROVED** [review]
Notes:
* Don't reference external variables (i.e. which are set by `build-package.sh`) outside of functions in `build.sh`. This is potential source of problems because `build.sh` scripts are subject for use by different tools, not only `build-package.sh` and thus must be self-sufficient.
* Don't override license installation step. You have `TERMUX_PKG_LICENSE_FILE`.
* Avoid hardcoded $TERMUX_PREFIX. Use `@TERMUX_PREFIX@` in patches - this value will be replaced automatically because patches are preprocessed.

**Comment by librehat** [comment]
Thanks a lot! I didn't see https://github.com/termux/termux-packages/wiki/Coding-guideline#patches 😓

**Pull request merged**